swtbot_test/src/utilities/Utility.java

```diff
@@ -142,11 +142,31 @@ public static void addComponentAndGenerate(String componentName) {
 		bot.sleep(15000);
 	}
 	
+	public static void removeComponentAndGenerate(String componentName) {
```

Comment:
E: separate this utility into two different utils for removing components and generating code.